### PR TITLE
Update photographers

### PR DESCRIPTION
**Pull request by jamesgorrie**
New list from Jo B / Robert H.

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala [hunk 1]
```diff
@@ -22,41 +22,85 @@ object PhotographersList {
...
+    "Graham Turner"  -> "The Guardian",
+    "Sean Smith"     -> "The Guardian",
+    "Jill Mead"      -> "The Guardian",
+    "Roger Tooth"    -> "The Guardian",
+    "James Mann"     -> "The Guardian",
+    "Rachel Vere"    -> "The Guardian",
+    "Alicia Canter"  -> "The Guardian",
+    "Bill Code"      -> "The Guardian",
+
+    // Past
```
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala]: Jo B has assured me that these people shouldn't come in through the FTP stuff, so we should be safe.

The alternative would be to move this list to the Picdar import, but then when things are uploaded, we might not get the benefit of adding this automagically.

:roller_coaster:

Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala]: As chatted with @theefer :kiss: might be the solution here.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala [hunk 2]
```diff
@@ -22,25 +22,72 @@ object PhotographersList {
...
+
+  val contractedPhotographers: Map[String, String] = Map(
+    "Antonio Zazueta"     -> "The Guardian",
+    "Christian Sinibaldi" -> "The Guardian",
     "Christopher Thomond" -> "The Guardian",
-    "Sean Smith"          -> "The Guardian",
     "David Levene"        -> "The Guardian",
-    "David Sillitoe"      -> "The Guardian",
-    "Eamonn Mccabe"       -> "The Guardian",
-    "Felicity Cloake"     -> "The Guardian",
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala]: She's gone altogether?

Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala]: Note that we have thoughts of her pics

Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala]: Just checked with Robert and she's a freelance writer for us (food). He's going to check the contracts.

Either way she doesn't seem to put herself as byline so we couldn't catch it with an automated filter. Instead, we may want to just do a batch override of her rights (where she's the byline through the picdar metadata overrides) and mark her as a contributor photographer. Someone would then have to do the same going forward when they import or use her pics.

**Comment by theefer** [comment]
Merging as Felicity doesn't belong in either staff or contractor list anyway.

**Comment by theefer** [comment]
Going to need to batch `/reindex` images matching these bylines.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala (hunk 2 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala]: We definitely have her in the [byline quite a lot](https://api.media.gutools.co.uk/images?q=byline%3A%22Felicity%20Cloake%22).

She is under the "contract photographers" In the Picdar copyright groups, so she will be picked up there.

There is only one that has been uploaded by us, and I suppose she would be flagged up there as a commissioned photographer (pushing the need for a "copy to metadata from usage rights" button)?

**Pull request merged**